### PR TITLE
Add GCP deploy docs

### PR DESCRIPTION
**Pull request by voron**
This PR adds initial doc on how to perform simple deploy of Polygon nodes in Google Cloud
Feel free to propose any changes here.

### CODE REVIEW

**Comment by victorcoder** [comment]
@voron It looks like the history is garbled, could you squash it or clean-up? thanks!

**Comment by ssandeep** [comment]
@victorcoder , I have asked @voron to raise a new PR as there was a force push on the master branch.

File: docs/validate/mainnet/install-gcp.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+* gcloud cli, local or [Cloud Shell](https://cloud.google.com/shell)
+* web console 
+
+We'll cover the first case in this manual. Let's start from deploy using CLI.
+1. Follow ["Before you begin" section](https://cloud.google.com/compute/docs/instances/create-start-instance#before-you-begin) to install and configure gcloud command-line tool. 
+Pay attention to default region and zone, choose ones closer to You or Your customers. You may use [gcping.com](https://gcping.com) to measure latency to choose closest location.
+2. Adjust the following command variables prior executing, when required
+   * `POLYGON_NETWORK` - choose `mainnet` or `mumbai` testnet network to run
+   * `POLYGON_NODETYPE` - choose `archive`,`fullnode` node type to run
+   * `POLYGON_BOOTSRAP_MODE` - choose bootstrap mode `snapshot` or `from_scratch`
```
[the remark this answers is not in the document]
Comment (voron) [reply docs/validate/mainnet/install-gcp.md]: fixed, PR rebased to current master

**Comment by voron** [comment]
@ssandeep please review it now

**Comment by victorcoder** [comment]
Much better now thx!

**Review by ssandeep — APPROVED** [review]

**Review by victorcoder — APPROVED** [review]
LGTM

**Pull request merged**